### ISSUE TITLE
[2022-02-05 05:51:37.680] [renderer1] [error] [Extension Host] Failed to load message bundle for file c:\Users\mlwp\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-1.4.3\out\extension

### ISSUE THREAD

**Issue by mwoodpatrick** [microsoft/vscode-edge-devtools#865]
**Environment (please complete the following information):**
 - OS Windows 11 
```
 - Version: 1.64.0 (user setup)
Commit: 5554b12acf27056905806867f251c859323ff7e9
Date: 2022-02-03T04:22:20.678Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.22000
```
 - Extension version 1.4.3

**Describe the bug:**
In debugging an issue loading a vscode test extension I was reviewing the "Window" logs using the "Developer: Open Log File and saw

```
[2022-02-05 06:45:53.049] [renderer1] [error] [Extension Host] Failed to load message bundle for file c:\Users\mlwp\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-1.4.3\out\extension
[2022-02-05 06:45:53.050] [renderer1] [error] [Extension Host] Failed to load message bundle for file c:\Users\mlwp\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-1.4.3\out\extension
[2022-02-05 06:45:53.051] [renderer1] [error] [Extension Host] Failed to load message bundle for file c:\Users\mlwp\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-1.4.3\out\extension
[2022-02-05 06:45:53.052] [renderer1] [error] [Extension Host] Failed to load message bundle for file c:\Users\mlwp\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-1.4.3\out\extension
[2022-02-05 06:45:53.053] [renderer1] [error] [Extension Host] Failed to load message bundle for file c:\Users\mlwp\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-1.4.3\out\extension
[2022-02-05 06:45:53.054] [renderer1] [error] [Extension Host] Failed to load message bundle for file c:\Users\mlwp\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-1.4.3\out\extension
[2022-02-05 06:45:53.054] [renderer1] [error] [Extension Host] Failed to load message bundle for file c:\Users\mlwp\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-1.4.3\out\extension
[2022-02-05 06:45:53.056] [renderer1] [error] [Extension Host] Failed to load message bundle for file c:\Users\mlwp\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-1.4.3\out\extension
```

**Repro steps:**
Just load vscode and review above log

**Expected behavior:**
Errors should not be ocurring

**Additional context:**

**Comment by mwoodpatrick** [comment]
Note I do see the file

`C:\Users\mlwp\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-1.4.3\out\extension.js`

Reinstalling extension did not help

**Comment by captainbrosset** [comment]
Thank you for filing this @mwoodpatrick. I'll open a task on the team's internal backlog so we can investigate and fix this.

**Comment by hkal** [comment]
This issue should be fixed in 1.4.5 (#888). Re-open if that is not the case.

**Comment by philsherry** [comment]
I’m doing some extension admin over the Easter break and noticed this same thing. Uninstalled, reinstalled, still happening. Did a search, ended up here, saw the note about re-opening if it isn’t fixed.
```
2024-03-30 18:49:24.878 [error] [Extension Host] Failed to load message bundle for file /Users/philsherry/.vscode/extensions/ms-edgedevtools.vscode-edge-devtools-2.1.5/out/extension
2024-03-30 18:49:24.953 [error] [Extension Host] Failed to load message bundle for file /Users/philsherry/.vscode/extensions/ms-edgedevtools.vscode-edge-devtools-2.1.5/out/extension
2024-03-30 18:49:24.957 [error] [Extension Host] Failed to load message bundle for file /Users/philsherry/.vscode/extensions/ms-edgedevtools.vscode-edge-devtools-2.1.5/out/extension
2024-03-30 18:49:24.959 [error] [Extension Host] Failed to load message bundle for file /Users/philsherry/.vscode/extensions/ms-edgedevtools.vscode-edge-devtools-2.1.5/out/extension
2024-03-30 18:49:24.961 [error] [Extension Host] Failed to load message bundle for file /Users/philsherry/.vscode/extensions/ms-edgedevtools.vscode-edge-devtools-2.1.5/out/extension
2024-03-30 18:49:24.965 [error] [Extension Host] Failed to load message bundle for file /Users/philsherry/.vscode/extensions/ms-edgedevtools.vscode-edge-devtools-2.1.5/out/extension
2024-03-30 18:49:24.966 [error] [Extension Host] Failed to load message bundle for file /Users/philsherry/.vscode/extensions/ms-edgedevtools.vscode-edge-devtools-2.1.5/out/extension
2024-03-30 18:49:24.968 [error] [Extension Host] Failed to load message bundle for file /Users/philsherry/.vscode/extensions/ms-edgedevtools.vscode-edge-devtools-2.1.5/out/extension
```
